runtime/expr/ztests/cut-dup-fields.yaml

```diff
@@ -3,6 +3,7 @@ script: |
   ! zq -z "cut rec.sub1,rec.sub1" in.zson
   ! zq -z "cut rec.sub,rec.sub.sub" in.zson
   ! zq -z "cut rec.sub.sub,rec.sub" in.zson
+  # XXX Add runtime evaluatable cut (i.e., dynamic path).
```

Comment:
Do this now?

Reply:
We already have a test for this compiler/ztests/dynamic-field-cut